browser-test/src/support/applicant_questions.ts

```diff
@@ -741,6 +741,49 @@ export class ApplicantQuestions {
     }
   }
 
+  async expectCommonIntakeConfirmationPageNorthStar(
+    wantUpsell: boolean,
+    wantTrustedIntermediary: boolean,
+    wantEligiblePrograms: string[],
+  ) {
+    if (wantTrustedIntermediary) {
+      await expect(
+        this.page.getByRole('heading', {
+          name: 'Programs your client may qualify for',
+        }),
+      ).toBeVisible()
+    } else {
+      await expect(
+        this.page.getByRole('heading', {name: 'Programs you may qualify for'}),
+      ).toBeVisible()
+    }
+
+    if (wantUpsell) {
+      await expect(
+        this.page.getByRole('heading', {
```

Comment:
**suggestion:** Since these check the same element, I think it would be worth creating a variable for this element (`this.page.getByRole('heading', {name: 'Create an account to save your application information',})`) then doing the if / else.

Reply:
Excellent point! Done.